qutebrowser/config/configfiles.py

```diff
@@ -647,6 +647,7 @@ def read_config_py(filename: str, raising: bool = False) -> None:
 
     if api.errors:
         raise configexc.ConfigFileErrors('config.py', api.errors)
+    config.instance.config_py_loaded = True
```

Comment:
This means it isn't set when there were errors - but in that case, the `config.py` is still loaded, just with some errors. This should probably be done before the `if api.errors:`.

Reply:
Where would you prefer it to be instead? After `api.finalize()`?

Reply:
Sounds good! :+1:  Then it'll be called regardless whether there were errors or not.